src/InertialSense.cpp

```diff
@@ -1355,7 +1354,7 @@ bool InertialSense::OpenSerialPorts(const char* portPattern, int baudRate)
         if (device->hasDeviceInfo()) {
             device->GetData(DID_SYS_PARAMS);
             device->GetData(DID_FLASH_CONFIG);
-            // device->GetData(DID_EVB_FLASH_CFG);
+            device->GetData(DID_GPX_FLASH_CFG);
```

Comment:
I believe there is a flag on ISDevice that indicates if it or has a GPX.
This slaos doesn't request DID_GPX_STATUS, which it should, but ideally, only if its is actually an IMX or the IMX has a GPX attached.